spec/requests/complete_booking_request_spec.rb

```diff
@@ -21,6 +21,7 @@
         }
       }
 
+      allow(BookingLocations).to receive(:find).and_return(double(:booking_location, name_for: ''))
```

Comment:
If the `location_id` this spec refers to is Hackney this is probably not necessary.